V2/time_tracker/time_entries/_domain/__init__.py

```diff
@@ -4,5 +4,6 @@
 from ._services import TimeEntryService
 from ._use_cases import (
     CreateTimeEntryUseCase,
-    DeleteTimeEntryUseCase
+    DeleteTimeEntryUseCase,
+    UpdateTimeEntryUseCase
```

Comment:
Try to add a comma after the last item of a list so github only updates the lines with changes